Code/GraphMol/MolStandardize/Tautomer.h

```diff
@@ -72,27 +281,58 @@ class RDKIT_MOLSTANDARDIZE_EXPORT TautomerEnumerator {
     "acceptor")
 
   */
-  std::vector<ROMOL_SPTR> enumerate(
-      const ROMol &mol, boost::dynamic_bitset<> *modifiedAtoms = nullptr,
-      boost::dynamic_bitset<> *modifiedBonds = nullptr) const;
-
-  //! returns the canonical tautomer from a set of possible tautomers
-  /*!
-    Note that the canonical tautomer is very likely not the most stable tautomer
-    for any given conditions. The default scoring rules are designed to produce
-    "reasonable" tautomers, but the primary concern is that the results are
-    canonical: you always get the same canonical tautomer for a molecule
-    regardless of what the input tautomer or atom ordering were.
+  TautomerEnumeratorResult enumerate(const ROMol &mol,
+                                     bool reassignStereo = true) const;
```

Comment:
Looks to me that it would be good to add `reassignStereo`  to `CleanupParameters`

Reply:
@jones-gareth Hi Gareth, the reason why I decided not to do that is that it really only affects the enumeration, not the canonicalisation (the canonical tautomer is reassigned stereo anyway). My thinking was that adding it to the `CleanupParameters` could be confusing. However I am happy to revisit this, particularly as it defaults to `true`. @greglandrum What are your thoughts on this?

Reply:
Since we have the other tautomer options in `CleanupParameters` I think it's consistent to include `tautomerReassignStereo` in `CleanupParameters`

Reply:
@greglandrum Thanks Greg, will do that and remove it from `enumerate()` then.